deployments/helm-chart/crds/transportserver.yaml

```diff
@@ -1,10 +1,11 @@
-{{- if .Values.controller.enableCustomResources }}
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: transportservers.k8s.nginx.org
   labels:
-    {{- include "nginx-ingress.labels" . | nindent 4 }}
+    app.kubernetes.io/name: "nginx-ingress"
```

Comment:
I guess it doesn't make sense that this is somehow a parameter right?

Reply:
Well from my reading of: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/ I understand that templating in `/crds` is not permitted.
> "These CRDs are not templated, but will be installed by default when running a helm install for the chart. If the CRD already exists, it will be skipped with a warning."


Additionally, `nginx-ingress.labels` includes release specific labels. I don't believe these make much sense anymore. Helm does no life-cycle changes past initial install. 
Upgrades and deletion of CRDs must be done via `kubectl`.

Reply:
Thanks! makes sense